src/reconciler.tsx

```diff
@@ -111,35 +147,65 @@ export function forceResize() {
 let catalogue: ObjectHash = {}
 export const extend = (objects: object): void => void (catalogue = { ...catalogue, ...objects })
 
-export function applyProps(instance: any, newProps: any, oldProps: any = {}, accumulative: boolean = false) {
+export function applyProps(instance: any, newProps: any, oldProps: any = {}, accumulative = false) {
   // Filter equals, events and reserved props
   const container = instance.__container
-  const { sameProps, handlers } = Object.keys(newProps).reduce(
-    (acc, key) => {
-      if (is.equ(newProps[key], oldProps[key])) acc.sameProps.push(key)
-
-      // Event-handlers ...
-      //   are functions, that
-      //   start with "on", and
-      //   contain the name "Pointer", "Click", "ContextMenu", or "Wheel"
-      if (is.fun(newProps[key]) && key.startsWith('on')) {
-        if (key.includes('Pointer') || key.includes('Click') || key.includes('ContextMenu') || key.includes('Wheel')) {
-          acc.handlers.push(key)
-        }
+
+  const sameProps = [] as string[]
+  const handlers = [] as string[]
+
+  let i,
+    keys = Object.keys(newProps)
```

Comment:
We should avoid chained assignment like this. It is difficult to read and (in my opinion) doesn't mesh well with prettier or a preference for no semi colons.

```suggestion
  let i
  let keys = Object.keys(newProps)
```

Reply:
https://eslint.org/docs/rules/no-multi-assign